app/helpers/operations_helper.rb

```diff
@@ -45,6 +45,11 @@ def format_operations_field(operation, field)
           link_to(t("operation.result_link_text.#{operation.kind}"),
             rails_blob_path(operation.attachment, disposition: "attachment"))
         end
+      when :pending
+        if operation.status == :pending && can?(:destroy, Operation)
```

Comment:
I believe you can do `can?(:destroy, operation)` (lowercase o) to DRY up this check for :pending